airbyte-webapp/src/components/CreateConnectionContent/CreateConnectionContent.tsx

```diff
@@ -104,10 +91,7 @@ const CreateConnectionContent: React.FC<CreateConnectionContentProps> = ({
     const job = LogsRequestError.extractJobInfo(schemaErrorStatus);
     return (
       <ContentCard>
-        <TryAfterErrorBlock
-          onClick={onDiscoverSchema}
-          additionControl={<SkipButton>{additionBottomControls}</SkipButton>}
```

Comment:
I believe this was removed because there were never any `additionBottomControls` passed in, correct?

Reply:
Correct!